src/ui/ui_render/ui_render_mixin.js

```diff
@@ -30,6 +31,38 @@ export function uiRenderMixin(kbnServer, server, config) {
   // render all views from ./views
   server.setupViews(resolve(__dirname, 'views'));
 
+  server.route({
+    path: '/bundles/app/{id}/bootstrap.js',
+    method: 'GET',
```

Comment:
Make sure this gets `config: { auth: false }`

Reply:
Good call. I pushed this up and merged with the latest master.